app.toml.template

```diff
@@ -0,0 +1,41 @@
+[App]
+name = "ai.h2o.llmstudio"
+Version = "{{VERSION}}"
+Title = "H2O LLM Studio"
+Description =  "A framework and no-code GUI designed for fine-tuning state-of-the-art large language models (LLMs)"
+LongDescription = "about.md"
+Tags = ["GENERATIVE_AI", "NLP", "DATA_SCIENCE"]
+
+[Runtime]
+# Must be installed in the server config
+RuntimeVersion = "llm_studio"
```

Comment:
i think we don't need this special runtime anymore, default runtime should work?